app/views/no-results.nunjucks

```diff
@@ -0,0 +1,18 @@
+{% extends 'layout.nunjucks' %}
+
+{% block pageTitle %}All service results{% endblock %}
```

Comment:
We should take a look what the SEO best practice for a results page is and maybe add an issue.